modules/nw-multus-remove-pod.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-remove-pod_{context}"]
+= Removing a Pod from an additional network
+
+You can remove a Pod from an additional network.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster.
```

Comment:
You need to have configured an additional network.
Unless this happens automatically, you also need to have added the pods to the network.